app/main/views/g_cloud.py

```diff
@@ -513,16 +513,33 @@ def view_project(framework_family, project_id):
         "data_value": current_project_stage
     }]
 
-    return render_template('direct-award/view-project.html',
-                           framework=framework,
-                           project=project,
-                           custom_dimensions=custom_dimensions,
-                           search=search,
-                           buyer_search_page_url=buyer_search_page_url,
-                           search_summary_sentence=search_summary_sentence,
-                           framework_urls=framework_urls,
-                           call_off_contract_url=framework_urls['call_off_contract_url'],
-                           project_outcome_label=project_outcome_label)
+    try:
+        following_framework = framework_helpers.get_framework_or_500(
+            data_api_client,
+            content_loader.get_metadata(framework['slug'], 'following_framework', 'slug'),
```

Comment:
Do we need to have this as defined metadata or could we safely derive it as the next framework in the same family from the database?

Eh, we could do it on the framework id but it's not super safe. I guess leave this as-is. 🤔

Reply:
Yeah, I thought about this a little. It would _probably_ be fine to derive it. But maybe not always. In the past we allowed suppliers to only have to apply to every other G-Cloud and two G-Clouds were live at the same time. This scenario would break `just pick the next framework iteration` model.